types/ember-cli-fastboot/index.d.ts

```diff
@@ -0,0 +1,16 @@
+// Type definitions for ember-cli-fastboot 2.2
+// Project: https://github.com/ember-fastboot/ember-cli-fastboot
+// Definitions by: Henry Majoros <https://github.com/hmajoros>
+//                 Dan Freeman <https://github.com/dfreeman>
+//                 Chris Krycho <https://github.com/chriskrycho>
+//                 James C. Davis <https://github.com/jamescdavis>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 4.4
+
+import { FastBoot as FastBootService } from './services/fastboot';
```

Comment:
Just did some poking, and the global Fastboot appears to be a different value than the service. See docs [here](https://github.com/ember-fastboot/ember-cli-fastboot/tree/master/packages/fastboot#default-globals).

---

For others following along, we worked through this and found:

- `Fastboot` is defined [here](https://github.com/ember-fastboot/ember-cli-fastboot/blob/909c1417ceeb8a5c92bb952b0d59be02bfdca546/packages/fastboot/src/ember-app.js#L110)
- That in turn is set up on the global [here](https://github.com/ember-fastboot/ember-cli-fastboot/blob/909c1417ceeb8a5c92bb952b0d59be02bfdca546/packages/fastboot/src/sandbox.js#L12)